common/cross-profiles/aarch64.sh

```diff
@@ -7,3 +7,5 @@ XBPS_CXXFLAGS="$XBPS_CFLAGS"
 XBPS_CROSS_CFLAGS="-march=armv8-a"
 XBPS_CROSS_CXXFLAGS="$XBPS_CROSS_CFLAGS"
 XBPS_CROSS_FFLAGS=""
+RUSTFLAGS="--sysroot=${XBPS_CROSS_BASE}/usr"
+RUST_TARGET="${XBPS_CROSS_TRIPLET}"
```

Comment:
should be aarch64-unknown-linux-gnu, XBPS_CROSS_TRIPLET is aarch64-linux-gnu

Reply:
Oh, overlooked that, thanks